guides/handGrenades.md

```diff
@@ -0,0 +1,107 @@
+It is best practice to exhaust all other means of disabling a vehicle before attempting to throw a frag.
+
+#### Buildings 
+
+Fragmentation and flashbang grenades are most effective when used in enclosed spaces due to the overpressure of the enclosed area 
+along with the shrapnel caused by the fragmentation. A flashbang or fragmentation grenade should be chosen accordingly based on whether
+on-combatants are inside of the room/building. If they are, a flashbang is preferred.
+
+### Grenade Types
+
+- Fragmentation (M67) 
+    - Also known as 'frags'. 
+    - Fuze time of 4 - 5 seconds.
+
+- Flashbang (M84)
+    - Also known as a 'flash'. 
+    - Fuze time of 1 - 2 seconds.
+
+- Smoke (M18)
+    - Comes in a variety of colors. White is typically used as a screen, while colored smokes are used to mark positions.
+    - Fuze time of 1 - 2 seconds.
+    - Lasts 60 seconds
+
+- Chemlights
+    - Deploys immediately 
+    - Used to mark cleared rooms or cleared areas at nighttime. 
+
+### ACE3 Grenades 
+
+The ACE3 grenade system endeavors to make grenades easier to use in Arma3. 
+Two additional keybinds are added in two different modes to assist with grenade usage, as well as a HUD element.
+
+#### ACE3 Grenades Keybinds
+
+- Shift + G
+    - Pulls up the ACE grenade context HUD. You will see the current grenade type, as well as a small keymap along with multiple dots in an arc fashion.
+    - The dots represent the path of the grenade.
+    - If a dot is green it means the grenade is expected to his the ground at the green dot.
+    - If a dot is yellow it means the dot will encounter be going through a closed space (like a window or doorway) and may hit a ledge and change direction.
+   ![Good grenade](grenadeImages/goodNadeWindow.jpg) 
+    - If a dot is red, it means that at that dot (distance) the grenade will encounter a hard object and not be able to pass (like a wall).
+   ![Bad Nade](grenadeImages/badNade.jpg) 
+
+    - Keybinds from here:
+        - Shift + G - Continue cycling through grenades 
+        - LMB - Throw the grenade 
+        - RMB - Cancel and back out of the menue 
+        - MMB (Scroll) - Change the arc/throw distance. 
+        - MMB (Click) - Pull the pin and release the spoon, 'cooking' the grenade. This is very dangerous and the operator should be aware of the grenades approximate fuze times.
+        - Ctrl + G - Changes the throw mode to a drop. Useful if you have a broken window and just want to drop the grenade on the other side. Or if you want to drop the grenade down a hatch at your feet. 
+
+#### ACE3 Grenades Usage 
+
+Operators should always endeavor to use the ACE3 Grenades contextual menu as it allows for more accurate employment of the weapons.
+If throwing the grenade through a window, the operator should ensure the window is broken first. Otherwise the grenade may not go through and instead bounce back at the operator's feet. 
```

Comment:
```suggestion
When throwing a grenade through a window, the operator should ensure the window is broken first. Otherwise the grenade will not go through and instead bounce back at the operator's feet. 
```
We could add a mod for window breaking, but that is not a priority at this time